src/Aspire.Dashboard/Components/Pages/Resources.razor.cs

```diff
@@ -421,7 +421,7 @@ private static string GetEndpointsTooltip(ResourceViewModel resource)
 
         if (displayedEndpoints.Count == 0)
         {
-            return "&ndash;";
+            return string.Empty;
```

Comment:
IMO if there is no content then don't display a tooltip rather than display a tooltip with a dash in it.